File: Certesian_Product/certesian_product.py
def certesian_product(arr1,arr2):
    result = []
    for i in arr1:
        for j in arr2:
            result.append([i, j])
    return result

list1 = [1,2]
list2 = [3,4,5]
print(certesian_product(list1, list2))

File: Algorithm/Certesian_Product/certesian_product.js
const certesian_product = (arr1, arr2) => {
    const ressult = [];
    for (i = 0; i < arr1.length; i++) {
        for (j = 0; j < arr2.length; j++) {
            ressult.push([ arr1[ i ], arr2[ j ] ]);
        }
    }
    return ressult;
};

const list1 = [ 1, 2 ];
const list2 = [ 3, 4 ];
console.log(certesian_product(list1, list2));

// Big-O = O(mn)